bot.py

```diff
@@ -99,32 +98,37 @@ async def send_message(self, destination, content=None, *args, **kwargs):
             if not kwargs.pop("allow_everyone", None):
                 content = content.replace("@everyone", "@ everyone").replace("@here", "@ here")
 
-        return await super().send_message(destination, content, *args, **kwargs)
+        return await destination.send(content, *args, **kwargs)
 
     async def send_file(self, destination, fp, *, filename=None, content=None, tts=False):
-        """ Override send_file to notify the server when an attachment could not be sent. """
+        """ Override send_file to notify the guild when an attachment could not be sent. """
         try:
-            return await super().send_file(destination, fp, filename=filename, content=content, tts=tts)
+            return await destination.send(content=content, tts=tts,
+                                          file=discord.File(fp, filename=filename))
         except discord.errors.Forbidden:
             return await self.send_message(destination, "**I don't have the permissions to send my attachment.**")
 
     async def delete_message(self, message):
         """ Override to add info on the last deleted message. """
         self.last_deleted_messages = [message]
-        await super().delete_message(message)
+        await discord.Message.delete(message)
```

Comment:
use method of the message instance instead of the class method

```suggestion
        await message.delete()
```